docs/UserGuide.md

```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
 
 ### 3.2 Application Home Window Breakdown
 
-### 3.3 Client Explanation
+Here is an overview of the different elements that makes up the Home Window of DonnaFin
 
-### 3.4 Application Client Window Breakdown
+![MainWindowScreenshot.png](./images/MainWindowScreenshot.png)
 
-### 3.5 Command Format
+#### 3.2.1 Quick Access Buttons
+* This area gives you access to the File button that allows the you to quickly exit the app, and the help 
+  button that redirects you to the User Guide for further guidance.
+
+#### 3.2.2 Command Box
+* The command box is where you can enter your commands in.
+
+#### 3.2.3 Results Display
+* The results display is where the action taken by DonnaFin in response to the command you keyed in is displayed.
+* In the event you enter an invalid or wrongly formatted command, the error message will be displayed here as well.
+
+#### 3.2.4 Client List Window
+In this section, you will be able to view a list of all the clients that have their records stored on DonnaFin. The index
+of a client corresponds to the index needed to be inputted in the `view` command. For example: if the client list is as
+per the image above, inputting the command `view 2` would take you to the client view of Bernice Yu, as she has the index
+of 2.
+
+The following attributes of your client is also viewable from the Client List Window:
+1. `Name`
+2. `Phone`
+3. `Address`
+4. `Email Address`
+
+### 3.3 Application Client Window Breakdown
+The Client window is accessed when you use the `view` command correctly. Though the client window shows content different
+from the main window, the `Quick Access Button`, `Command Box` and `Results Display` are all the same as explained 
+earlier in section 3.2. 
+
+<div markdown="block" class="alert alert-warning">:exclamation: **Caution:**
+
+Do take note that the `Command Box` is tab specific in the Client Window 
+so do take not that you are inputting the correct command in the correct tab.
+
+</div>
+
+Here is a picture showing all the common elements spread accross all tabs in the Client View. 
+
+![ClientView](images/ClientView.png)
+
+#### 3.3.1 Client Tab Buttons
+* These buttons are specific to the Client Window, and you can use these buttons to toggle through the different tabs
+that have been implemented to access and edit client information. The tabs are as follows:
+  1. `Contact`
+  2. `Policies`
+  3. `Assets`
+  4. `Liabilities`
+  5. `Notes`
+    
+The tabs can also be toggled through using the `tab` command as shown [here.](https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html#431-switching-tabs-in-client-window-tab)
+
+#### 3.3.2 Contact Tab
+The `Contact Tab` is the Client View tab you first see when transitioning from the Home View to the Client View.
```

Comment:
it should not be in code fences here tbh, like all other uses of home view, client view, this is just a name for a component, not a class name or object or CLI usage etc.